docs/docs/http-api.mdx

```diff
@@ -66,6 +66,21 @@ rasa run \
     --auth-token thisismysecret
 ```
 
+or via environment variable:
+:::tip Security best practice
+
+We recommend that you use environment variables to store
+sensitive information such as tokens and secrets
+as they will not be stored in your shell history.
+
+:::
+
+```bash
+export AUTH_TOKEN=thisismysecret
```

Comment:
Please ignore these words in the typo CI yaml file found at the root of the project.